### PR TITLE
ISSD-2814 [Testray] Add logic to not add filter to URL when modal

### PR DESCRIPTION
**Pull request by rafaelfs98**

### CODE REVIEW

**Comment by rafaelfs98** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

20 files changed, 407 insertions(+), 16 deletions(-)

**Comment by rafaelfs98** [comment]
ci:test:sf

**Comment by rafaelfs98** [comment]
@joseabelenda  or @kevenleone

**Comment by rafaelfs98** [comment]
ci:test:sf

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/ManagementToolbar/ManagementToolbarResultsBar.tsx [hunk 1]
```diff
@@ -30,8 +34,33 @@ const ManagementToolbarResultsBar: React.FC<ManagementToolbarResultsBarProps> =
 		dispatch({payload: null, type: ListViewTypes.SET_CLEAR});
 	};
 
+	const handleRemoveItemDoFilter = (itemToRemove: string) => {
```
Comment (joseabelenda) [inline workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/ManagementToolbar/ManagementToolbarResultsBar.tsx:37]: Rename.
Please use English instead Portuguese.

**Review by joseabelenda — CHANGES_REQUESTED** [review]

**Comment by joseabelenda** [comment]
resent:https://github.com/is-solutions-delivery/liferay-portal/pull/5468

**Comment by joseabelenda** [comment]
@rafaelfs98 see my changes
https://github.com/is-solutions-delivery/liferay-portal/pull/5468/commits/4e08a1e676990fefe9090c8645d6b2e6ce85d5b1

**Review by kevenleone — CHANGES_REQUESTED** [review]
Hey guys @joseabelenda @rafaelfs98 

Sorry for reopening this issue, is just because I forgot the "send" my pending review here.

@rafaelfs98 this looks really nice, good job!

please see my suggestions and analyze if it makes sense for you.

Only thing I disliked is this `isModal` implementation.

For me, ListView should not know about custom logic for Modals, it should be 100% transparent to any use-case.

You can try to find a better name about what you are doing for filters.

Try something like: `ignoreFilters`, `applyFilters: false` (default true).

Something like, but please without`isModal`

You can follow-up this changes here: https://github.com/is-solutions-delivery/liferay-portal/pull/5469

**Comment by kevenleone** [comment]
You can close after reviewing my suggestion

**Pull request closed without merging**